api/ReadableStreamBYOBReader.json

```
@@ -213,6 +213,55 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "options_min_parameter": {
+          "__compat": {
+            "description": "<code>options.min</code> parameter",
```

We could try an `impl_url` with a text fragment here:

```suggestion
            "description": "<code>options.min</code> parameter",
            "impl_url: "https://streams.spec.whatwg.org/#:~:text={%20min%20}",
```